.travis.yml

```diff
@@ -9,7 +9,7 @@ before_install:
 - dep ensure
 
 install:
-- go get -t -d ./...
+- go get -t -d ./pkg/controllers/v1alpha1/...
```

Comment:
This just builds the code, right? The new controller should at least compile. :)

Reply:
Good catch! Just fixed it.